### ISSUE TITLE
While upgrading a package, una ran autoremove without prompting first

### ISSUE THREAD

**Issue by NinjaCheetah** [AFK-OS/una#27]
Essentially what the title said. I ran `una update` and then `una upgrade`, and at the end of the upgrade, it uninstalled all the older kernel versions that I keep installed. Not sure if this is intended behavior or not.

It seems like maybe it ran `autoremove`? Either way I would prefer that it *asks* before doing anything other than what I told it to do.

**Comment by ghost** [comment]
yes, it ran autoremove, it surprised me too, can you update the title with that?

**Comment by ghost** [comment]
It's this line the culprit `sudo apt-get dist-upgrade -y --auto-remove` At a minimum you want to remove that `-y` and `--auto-remove` also seams unnecessary.

**Comment by ghost** [comment]
a bit lower, he's doing it again with  `sudo apt-get purge -y "${pkgname}"` in _remove()

**Comment by NinjaCheetah** [comment]
Yeah I agree that it shouldn't be running anything with `-y`. I've since stopped using una because I only was using it for one package and it's available as a Flatpak anyway but it would still be nice to see that changed.

**Comment by ghost** [comment]
it's not even a year old.... I would have put a -beta in the name to properly manage expectations. Weird stuff like this will slowly go away.
